bigchaindb/backend/mongodb/connection.py

```diff
@@ -113,6 +113,8 @@ def _connect(self):
                                              replicaset=self.replicaset,
                                              serverselectiontimeoutms=self.connection_timeout,
                                              ssl=self.ssl)
+                if self.login is not None and self.password is not None:
```

Comment:
I wonder if we should raise an exception if they set one of `self.login` or `self.password`, but not the other?